styles/components/_form.scss

```diff
@@ -1,4 +1,5 @@
-form {
+form,
+.form {
```

Comment:
There are various places where inputs need form styling, but aren't in forms, the .form class handles that situation.